felix/wireguard/wireguard_test.go

```diff
@@ -289,8 +290,8 @@ func describeEnableTests(enableV4, enableV6 bool) {
 			rule.Priority = rulePriority
 			rule.Table = tableIndex
 			rule.Invert = true
-			rule.Mark = firewallMark
```

Comment:
if you change `firewallMark` to unit32 in definition, there are less uint32 cast to do below. You do need to cast to int when assigning to `WireguardFirewallMark`.